Doc/library/stdtypes.rst

```diff
@@ -1376,6 +1376,108 @@ objects that compare equal might have different :attr:`~range.start`,
    single: str (built-in class); (see also string)
    object: string
 
+.. _text-methods-summary:
+
+Text and Binary Sequence Type Methods Summary
+=============================================
+The following table summarizes the text and binary sequence types methods by
+category.
+
+
++--------------------------+------------------------------+------------------------------+
+| Category                 |  str methods                 |  bytes methods               |
++==========================+==============================+==============================+
+| Formatting               |  :meth:`str.format`          |                              |
++                          +------------------------------+------------------------------+
+|                          |  :meth:`str.format_map`      |                              |
```

Comment:
Maybe ``__mod__`` (``%``) should be included here too.

Reply:
I've added some lines for the printf-style and for f-strings.  I think I linked them to the best places for a description.